roles/generate-docs/tasks/create_readme.yml

```diff
@@ -0,0 +1,12 @@
+---
+# Attempted to generate docs with the uri module like:
+# https://github.com/redhat-cop/infra-ansible/blob/master/roles/ansible/tower/manage-inventories/tasks/process-group.yml
+# But recieved error "unautorized". Is this method specifically for ansible tower?
+
+# Also, attempted using the pandoc cli tool like:
+# https://github.com/redhat-cop/infra-ansible/blob/master/roles/notifications/md-to-html/tasks/convert_md_to_html.yml
+# But I ran into issues with this as the template is pre-written in markdown
+
+- name: Convert template to html
+  shell: >
```

Comment:
Let's forgo the `shell` module and just store the output from the `lookup` call in a fact. This can then be used by other roles to either write files or upload to hosting sites using APIs.

Please make sure to document the name of the "output" fact in the role README.